nexus/db-queries/src/db/datastore/pub_test_utils.rs

```diff
@@ -12,32 +12,135 @@ use crate::authz;
 use crate::context::OpContext;
 use crate::db;
 use crate::db::DataStore;
-use dropshot::test_util::LogContext;
 use omicron_test_utils::dev::db::CockroachInstance;
+use slog::Logger;
 use std::sync::Arc;
 use uuid::Uuid;
 
+#[cfg(test)]
+mod test {
+    use super::*;
+    use nexus_test_utils::db::test_setup_database;
+
+    enum TestState {
```

Comment:
super nitpicky and obnoxious naming quibble (sorry): is this really a "state", or is it more of a "kind"? AFAICT, this is fixed when constructing the `TestDatabase`, and doesn't change at runtime --- "state" to me suggests that it might move between states?

Reply:
Patched in https://github.com/oxidecomputer/omicron/pull/6881/commits/0d59b623a97fc87c383f5fe4e745bddda019ef3f , `s/state/kind`